README.md

```diff
@@ -14,9 +14,9 @@ To find out if your company's tool is feasible for this approach, check out our
 We provide reference implementations that provide fully functional integrations with popular service desks. 
 These implementations, while functional, are examples only, and have not been vetted for production use.
 
-- [Genesys Cloud](./src/middleware/genesys)  
-- [Twilio Flex](./src/middleware/flex) 
-- [NICE inContact](./src/middleware/incontact)
+- [Genesys Cloud](./src/genesys/webChat/server)  
```

Comment:
Lets link just to `src/genesys/webChat` instead of plopping them a level down into the server. They need both! Same for the other ones.